### ISSUE TITLE
Panel block

### ISSUE THREAD

**Issue by VK-VZ** [UnamSanctam/UnamWebPanel#350]
Hello unam.  How can I find out if my panel and my IP are blocked? The hosting did not block anything, everything opens fine for me.  But the workers themselves will be added 10 times less.

**Comment by UnamSanctam** [comment]
You'd have to try and find it on IP block sites, though usually there aren't any large scale blocks, since all the users would need some sort of program that does it.

**Comment by VK-VZ** [comment]
Which programm?  Isn't that what Windows Defender does?

**Comment by UnamSanctam** [comment]
No, either they'd have to install some specific IP block program or some other more intrusive antivirus.

**Comment by VK-VZ** [comment]
I think less than 1% of people use such programs.  It turns out that it is safe to use the panel and in vain everyone is afraid and discusses blocking the panel.
Thanks for answering
